src/main/java/duke/models/assignedtasks/AssignedTaskWithPeriod.java

```diff
@@ -59,6 +59,35 @@ public AssignedTaskWithPeriod(int pid, int tid, boolean isDone,
         duration = Duration.between(getStartDate(), getEndDate()).toMillis();
     }
 
+
+    /**
+     * .
+     *
+     * @param pid          .
+     * @param tid          .
+     * @param isDone       .
+     * @param isRecurrsive .
+     * @param stime        .
+     * @param etime        .
+     * @param type         .
+     * @param uniqueId     .
+     */
+    public AssignedTaskWithPeriod(int pid, int tid, boolean isDone,
+                                  boolean isRecurrsive, String stime, String etime, String type, int uniqueId)
+            throws DukeException {
```

Comment:
Perhaps separate the parameters to 3 rows ?

Reply:
sure